src/libraries/System.Threading.RateLimiting/tests/SlidingWindowRateLimiterTests.cs

```diff
@@ -0,0 +1,638 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Threading.RateLimiting.Test
+{
+    public class SlidingWindowRateLimiterTests : BaseRateLimiterTests
```

Comment:
None of the tests actually make use the sliding window all the segment sizes are 1.